me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/AuthToken.java

```diff
@@ -3,18 +3,21 @@
 import java.io.Serializable;
 import java.util.Calendar;
 
-final class AuthToken implements Serializable {
-  private IDmeScope scope;
+import me.id.webverifylib.listener.IDmeScope;
+
+public final class AuthToken implements Serializable {
+  private String scopeId;
   private String accessToken;
   private String refreshToken;
-  private Calendar expiration;
+  private Calendar accessTokenExpiration;
+  private Calendar refreshTokenExpiration;
 
-  public IDmeScope getScope() {
-    return scope;
+  public String getScopeId() {
+    return scopeId;
   }
 
-  public void setScope(IDmeScope scope) {
-    this.scope = scope;
+  public void setScopeId(IDmeScope scope) {
```

Comment:
I think should be clear if the parameter is directly the `scopeId`. As it is now, it is a lil bit confusing for me. I would expect to be able to get back the same scope I passed to the token through this setter.